rvip/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(rvip)
+
+add_compile_options(-std=c++11)
+
+find_package(catkin REQUIRED COMPONENTS
+  tf
+  rospy
+  roscpp
+  roslint
+  pcl_ros
+  std_msgs
+  cv_bridge
+  sensor_msgs
+  geometry_msgs
+  image_transport
+  visualization_msgs
+)
+
+roslint_cpp()
+
+# DEPENDENCIES
+find_package(OpenGR REQUIRED PATHS /usr/local/lib/cmake)
+find_package(PCL 1.8 REQUIRED)
+
+include_directories(include ${catkin_INCLUDE_DIRS} ${PCL_INCLUDE_DIRS} ${OpenGR_INCLUDE_DIRS})
+
+link_directories(${PCL_LIBRARY_DIRS} )
+add_definitions(${PCL_DEFINITIONS} )
+
+catkin_package( CATKIN_DEPENDS
+  std_msgs
+  sensor_msgs
+  geometry_msgs
```

Comment:
Where are all the other dependencies?

Reply:
Updated. [See commit](https://github.com/cardboardcode/rvip-1/commit/78722a8fee690dad35aa1e37113efd79ee925b6f)